sim/side.ts

```diff
@@ -1072,7 +1072,9 @@ export class Side {
 
 	choosePass(): boolean | Side {
 		const index = this.getChoiceIndex(true);
-		if (index >= this.active.length) return false;
+		if (index >= this.active.length) {
+			return this.emitChoiceError(`Can't pass: You sent more choices than unfainted Pokémon`);
```

Comment:
It gets sent if the command is sent as `move 1 1, pass` and Tatsugiri is position p1a. As I said, I can change it to just return true, and that would ignore trailing `pass`'s comands. But we can't just leave it as false since it can lead to the game being stuck.

Reply:
There are two problems with that:

1. The prior function, `choose()`, already checks if the number of actions is greater than the number of unfainted Pokémon.

2. If I changed `>=` to `>`, it would cause an error later in the function, since there is no `active[2]`.

Basically, what happens is if the action doesn't start with `pass`, the function will just skip the action `move 1 1` over Tatsugiri because it is `commanding`, leaving an extra `pass` to handle in the end. However, it did pass the original check for the number of unfainted Pokémon. Even if I changed it to '>', it would still need an `index >= this.active.length` check to return `true`.

If you want, I can do a complex check to return `true` if `index == this.active.length && this.active.some(p => p.fainted || pokemon.volatiles['commanding'])`. I haven't tested it, but this probably also happens if the Pokémon on the left is fainted.

Reply:
1. Yes, the same issue occurs with fainted Pokémon on the left.
2. It can't be `index == this.active.length`, because the same issue happens in a Triple Battle when both the left and center Pokémon are fainted, and the command is `move 1 1, pass, pass`.

It's probably better to leave it as it is now, unless you want me to add a check—and return `true` or `this.emitChoiceError`—based on the number of active Pokémon in the game type.